Add a guides for generating embeddings with Mistral, OpenAI, Voyage, and Cloudflare (#2965)
meilisearch/documentation: guides/embedders/cloudflare.mdx

---
title: Semantic Search with Cloudflare Worker AI Embedding - Meilisearch documentation
description: This guide will walk you through the process of setting up Meilisearch with Cloudflare Worker AI embeddings to enable semantic search capabilities.
---

> # Semantic search with Cloudflare Worker AI embedding

Reviewer: ```suggestion
# Semantic search with Cloudflare Worker AI embeddings
```

## Introduction

This guide will walk you through the process of setting up Meilisearch with Cloudflare Worker AI embeddings to enable semantic search capabilities. By leveraging Meilisearch's AI features and Cloudflare Worker AI's embedding API, you can enhance your search experience and retrieve more relevant results.

## Requirements

To follow this guide, you'll need:

> - A [Meilisearch Cloud](https://www.meilisearch.com/cloud) project running version 1.10 or above with the Vector store activated.

Reviewer: ```suggestion
- A [Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=vector-search&utm_source=docs&utm_medium=cloudflare-embeddings-guide) project running version 1.10 or above with the Vector store activated.
```

- A Cloudflare account with access to Worker AI and an API key. You can sign up for a Cloudflare account at [Cloudflare](https://www.cloudflare.com/).
- Your Cloudflare account ID.
- No backend required.

## Setting up Meilisearch

> To set up an embedder in Meilisearch, you need to configure it to your settings. You can refer to the [Meilisearch documentation](https://www.meilisearch.com/docs/reference/api/settings#update-embedder-settings) for more details on updating the embedder settings.

Reviewer: ```suggestion
To set up an embedder in Meilisearch, you need to configure it to your settings. You can refer to the [Meilisearch documentation](/reference/api/settings?utm_campaign=vector-search&utm_source=docs&utm_medium=cloudflare-embeddings-guide#update-embedder-settings) for more details on updating the embedder settings.
```
